src/sprites.rs

```diff
@@ -0,0 +1,199 @@
+use ndarray::prelude::arr2;
+use ndarray::Array2;
+use ratatui::style::Color;
+
+#[derive(Debug, Clone)]
+#[allow(dead_code)]
+pub struct CharPixel {
+    pub char: char,
+    pub fg: Option<Color>,
+    pub bg: Option<Color>,
+}
+
+#[derive(Debug, Clone)]
+#[allow(dead_code)]
+pub struct Sprite {
+    pub chars: Array2<CharPixel>,
+}
+
+#[allow(dead_code)]
+const BOX_ELEMENTS: [char; 32] = [
+    '▀', '▁', '▂', '▃', '▄', '▅', '▆', '▇', '█', '▉', '▊', '▋', '▌', '▍', '▎', '▏',
+    '▐', '░', '▒', '▓', '▔', '▕', '▖', '▗', '▘', '▙', '▚', '▛', '▜', '▝', '▞', '▟',
+];
+
+#[allow(dead_code)]
+pub fn get_player_sprite_8_simple() -> Sprite {
```

Comment:
you may be able to make this `const` for compile time evaluation. This might depend on what is going on under the hood for `arr2`

Reply:
That'd be sweet, arr2 builds a numpy-like array object so there is a bunch of dynamic allocation going on unfort. No reason this couldn't be replaced with a fixed size slice and made const tho.